README.md

```diff
@@ -202,7 +202,8 @@ Just edit the issue title, description or comments as a regular buffer and use `
 | `OctoNvimPullAdditions`     | `DiffAdd`        |
 | `OctoNvimPullDeletions`     | `DiffDelete`     |
 | `OctoNvimPullModifications` | `DiffChange`     |
-| `OctoNvimEditable`          | `NormalFloat`.bg|
+| `OctoNvimEditable`          | `NormalFloat`.bg |
+| `OctoNvimOwned`             | `PmenuSel`       |
```

Comment:
If we finally use the bubble approach, this should be something like `OctoNvimBubbleAuthor`. BTW, It would be nice to get rid of the `Nvim` word in all these highlight group names

Feels so good to do a PR from octo ... :D

Reply:
Haha, amazing. :star_struck: 

Closing in regards of [this comment](https://github.com/pwntester/octo.nvim/pull/116#issuecomment-789578727).